server/@types/educationAndWorkPlanApi/index.d.ts

```diff
@@ -633,11 +667,6 @@ export interface components {
        * @description An optional ISO-8601 date representing the target completion date of the Goal.
        */
       targetCompletionDate: string
-      /**
-       * @example null
-       * @enum {string}
-       */
-      status: 'ACTIVE' | 'COMPLETED' | 'ARCHIVED'
```

Comment:
Now that the `status` field is removed from the `UpdateGoalRequest` API type, I think you're also going to need to remove it from `UpdateGoalDto` and `UpdateGoalForm`, and the associated mappers and test data builders
Sorry! 🤷‍♂️